certification/engine/engine.go

```diff
@@ -98,3 +113,25 @@ func AllChecks() []string {
 	}
 	return all
 }
+
+func OperatorPolicy() []string {
+	checks := make([]string, len(operatorPolicyChecks))
```

Comment:
Looks like this could be a small utility function. Keep the exported funcs, but use a utility func and pass in the list of checks.

Reply:
In a future PR, I'm planning to do something like func `GetChecks(policy string) []string.`  Was planning to knock it out once this merges.